packages/ibm-products/src/components/Guidebanner/GuidebannerElementButton.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * Copyright IBM Corp. 2023, 2023
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+// Import portions of React that are needed.
+import React from 'react';
+
+// Other standard imports.
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import { Button } from 'carbon-components-react';
+import { Crossroads16 } from '@carbon/icons-react';
+
+import { getDevtoolsProps } from '../../global/js/utils/devtools';
+import { pkg } from '../../settings';
+
+// Carbon and package components we use.
+/* TODO: @import(s) of carbon components and other package components. */
+
+// The block part of our conventional BEM class names (blockClass__E--M).
+const blockClass = `${pkg.prefix}--guidebanner__element-button`;
+const componentName = 'GuidebannerElementButton';
+
+/**
+ * One of two buttons styled specifically for the GuidebannerElement.
+ */
+const GuidebannerElementButton = ({ children, className, type, ...rest }) => {
+  if (type === 'crossroads') {
```

Comment:
I think we usually use `renderIcon` instead (see ActionBarItem and ButtonMenu for examples).

Reply:
Thanks, wasn't happy with the generic `type`.